activate.sh

```diff
@@ -24,7 +24,7 @@ cp -v $manifest_dir/private.yaml $kube_dir
 
 # Make sure that the controller node looks for the local pause image
 # TODO: remove this as soon as possible. As an idea, we could use a systemd drop-in unit.
-sed -i 's|--config=/etc/kubernetes/manifests|--config=/etc/kubernetes/manifests --pod-infra-container-image=sles12/pause:1.0.0|g' /etc/kubernetes/kubelet
+sed -i '/^KUBELET_ARGS=/ s/=.*/="--pod-manifest-path=\/etc\/kubernetes\/manifests --pod-infra-container-image=sles12\/pause:1.0.0"/' /etc/kubernetes/kubelet
```

Comment:
My proposal would be
```
sed -i 's|^KUBELET_ARGS="--pod-manifest-path=/etc/kubernetes/manifests|KUBELET_ARGS="--pod-infra-container-image=sles12/pause:1.0.0 --pod-manifest-path=/etc/kubernetes/manifests|' /etc/kubernetes/kubelet
```
this prevents from multi-run and multi-changes on this same line for kubelet config file